### ISSUE TITLE
offcputime large bogus numbers

### ISSUE THREAD

**Issue by brendangregg** [iovisor/bcc#704]
Testing on Linux 4.8-rc4:

```
# ./offcputime.py
[...]

    ffffffff810a95f1 finish_task_switch
    ffffffff8185d385 schedule
    ffffffff810a5a30 smpboot_thread_fn
    ffffffff810a2628 kthread
    ffffffff81861e1f ret_from_fork
    -                migration/3 (27)
        18446744073704068

```

I don't know where these big numbers are coming from. They are about 2 << 53.

**Comment by glommer** [comment]
Could it be a kernel bug? I have started seeing this myself after I have upgraded to 4.8.0-rc7. Haven't seen this before with the distro kernel I was running, although the workload is exactly the same.

**Comment by brendangregg** [comment]
@glommer yes, it's a kernel bug, discussed in #728 and raised on lkml today. Hopefully fixed in 4.9-rc1 I'd guess. I retested offcputime without the bug and it worked. Closing ticket.
